blueflood-core/src/main/java/com/rackspacecloud/blueflood/types/BluefloodEnumRollup.java

```diff
@@ -7,13 +10,22 @@
     private Map<Long,Long> hashedEnumValues2Count = new HashMap<Long, Long>();
 
     public BluefloodEnumRollup withEnumValue(String valueName, Long value) {
-        this.stringEnumValues2Count.put(valueName, value);
-        this.hashedEnumValues2Count.put((long) valueName.hashCode(), value);
-        return this;
+        long count = 0;
+        if (this.stringEnumValues2Count.containsKey(valueName)) {
+            count = this.stringEnumValues2Count.get(valueName);
+        }
+        this.stringEnumValues2Count.put(valueName, count+value);
+
+        return this.withHashedEnumValue((long)valueName.hashCode(), value);
     }
 
     public BluefloodEnumRollup withHashedEnumValue(Long hashedEnumValue, Long value) {
-        this.hashedEnumValues2Count.put(hashedEnumValue, value);
+        long hashcount = 0;
+        if (this.hashedEnumValues2Count.containsKey(hashedEnumValue)) {
+            hashcount = this.hashedEnumValues2Count.get(hashedEnumValue);
+        }
```

Comment:
@VinnyQ 

value is a bad name for this variable, it should be incomingCount or something to that effect. 
This test case explains why we need to add the existingCount with the incomingCount. 
https://github.com/rackerlabs/blueflood/pull/519/files#diff-437d2681df3fc60b82f4ed2954b87af2R37

In a single 5 min rollup period, if we see the enum value 'call_xyz_api:OK' first and again after say 2 mins 'call_xyz_api:OK', we want the map to store 'OK' => 2 and hash('OK') => 2. Is that right? If we do not add the 2 values, then the last OK will overwrite the first and the only value stored in the maps will be 'OK' => 1